esbuild.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const path = require('path');
+const { argv } = require('process');
+const NodeResolve = require('@esbuild-plugins/node-resolve');
+const esbuild = require('esbuild');
+const yargs = require('yargs');
+const { CreateStylePlugin } = require('./style-plugin');
+
+const src = './src';
+const argsObj = yargs(argv).argv;
+
+const electronEntryFiles = {
+    renderer: [path.resolve(__dirname, 'src/electron/views/renderer-initializer.ts')],
+    main: [path.resolve(__dirname, 'src/electron/main/main.ts')],
+};
+const unifiedOutdir = path.join(__dirname, 'extension/unifiedBundle');
+
+const webExtensionEntryFiles = {
+    background: `${src}/background/background-init.ts`,
+    injected: `${src}/injected/client-init.ts`,
```

Comment:
In webpack, the injected entry point was defined as using both `client-init.ts` and `stylesheet-init.ts` as roots. Is omitting that here intentional? I'd have expected to need an update folding `stylesheet-init` into `client-init`

Reply:
Good catch! I missed adding that to the client-init.ts file in this branch.